### PR TITLE
add zh CN locale

### PR DESCRIPTION
**Pull request by Rictusempra**
- I also correct some typos in en a little bit
- implement to #58

### CODE REVIEW

**Comment by Rictusempra** [comment]
And I'm a little confused why auto-merging is blocked

**Comment by RobinJadoul** [comment]
There hasn't been a review yet. Otherwise, it seems to be able to merge fine after that.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Keywi
 ===
 
-Minimalistic Keepass plugin using Web Extensions and KeepassHTTP.
+Minimalistic Keepass extension using Web Extensions and KeepassHTTP.
```
Comment (Rictusempra) [inline README.md]: Just a note, Keywi is an `extension`.
Plugins are: Flash, Silver Light, etc.

**Comment by LEDfan** [comment]
Thanks for your contribution. Can you please add a new translation:
```
 "initSSFailed": {
    "description": "Message shown when the secure storage can't be initialized",
   "message": "Failed to initialize Secure Storage!"
}
```

**Comment by Rictusempra** [comment]
@LEDfan since you haven't merged #72 yet, pls add this manually:

`
"initSSFailed": {  
"description": "当 Secure Storage 无法初始化时会提醒",  
"message": "Failed to initialize Secure Storage!"  
}
`

**Comment by RobinJadoul** [comment]
@Rictusempra It seems you translated the description instead of the message.

**Comment by Rictusempra** [comment]
Sorry, my fault. Edited

**Pull request merged**